.github/workflows/ci.yml

```diff
@@ -2,9 +2,15 @@ name: ci
 on: [push, pull_request]
 jobs:
   linux:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
```

Comment:
I think our CI builds have stopped running since this MR, github/gitlab yaml is not fun